src/content/getting-started/architecture/networkplugin-syncer/ovn-kubernetes/_index.en.md

```diff
@@ -8,28 +8,32 @@ A specific handler component is deployed for the
 [OVN](https://www.ovn.org/en/architecture/) is a project that builds on top
 of Open vSwitch providing a rich high level API for describing virtual
 network components like Logical Routers, Logical Switches, Load balancers,
-Logical Ports. OVN Kubernetes is a Cloud Management System Plugin (CMS plugin)
-in terms of the OVN project.
+Logical Ports. [OVN Kubernetes](https://github.com/ovn-org/ovn-kubernetes) is a Cloud Management System Plugin (CMS plugin)
+which manages OVN resources to setup networking for Kubernetes clusters.
 
 The OVN Kubernetes handler watches for Submariner `Endpoints` and Kubernetes
 `Nodes` and interfaces with the OVN databases (OVN NorthDB and SouthDB) to store and
-maintain information necessary for Submariner, including:
+create OVN resources necessary for Submariner, including:
 
 * A logical router named `submariner_router` that handles the communication
   to remote clusters and has a leg on the network which can talk to the
-  `ovn-k8s-gw0` interface on the Gateway node. This router is pinned to
+  `ovn-k8s-sub0` interface on the Gateway node. This router is pinned to
   the active Gateway chassis.
 
-* Routing policies added to the existing `ovn_cluster_router` which redirect
+* OVN Logical Router Static Routes added to the `submariner_router` to ensure
+  local traffic destined for remote clusters and remote traffic destined for local
+  resources is routed correctly.
+
+* OVN Logical Router Policies added to the existing `ovn_cluster_router` which redirect
   traffic targeted for remote routers through the `submariner_router`.
 
 * A `submariner_join` logical switch that connects the `submariner_router`
   with the `ovn_cluster_router`.
 
 ## The handler architecture
 
-The following diagram illustrates the OVN Kubernetes handler architecture where the
-blue elements represent the OVN Kubernetes native network elements and the yellow
-elements are introduced by Submariner.
+The following diagram illustrates the required Submariner OVN architecture overlayed
```

Comment:
Can you please copy the original image to the following PPT [*] and add the following comment which will help us to make further changes in future if required.
```
<!-- Image Source: https://docs.google.com/presentation/d/180CtHZnr9PP5Rh98VEmkQz3ovc5AGXG9wosoHMLhgaY/edit -->

```

[*] https://docs.google.com/presentation/d/180CtHZnr9PP5Rh98VEmkQz3ovc5AGXG9wosoHMLhgaY/edit#slide=id.g8e581e9edd_0_142